pandas/core/generic.py

```diff
@@ -6362,6 +6360,16 @@ def fillna(
         1   3.0 4.0 NaN 1
         2   NaN 1.0 NaN 5
         3   NaN 3.0 NaN 4
+
+        Replace with a DataFrame with the same column name
```

Comment:
I think this could be clarified - replacement happens along same column names and same indices. Maybe the example `df2` could also have a column with a different name, such as `columns=list("ABCE")` so it's clear that replacement only happens for the columns they have in common

Reply:
You are correct, now corrected